.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy Docker
+
+# Run workflow on tags starting with v (eg. v2, v1.2.0)
+on:
+  push:
+    tags:        
+      - v*
+
+jobs:
+  Deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: JM-Lemmi
+          password: ${{ secrets.GHCR_TOKEN }}
```

Comment:
I think you might be able to use
```
username: ${{ github.actor }}
password: ${{ secrets.GITHUB_TOKEN }}
```
if you don't want your own token to be used. But it works either way 🤷

Reply:
I didn't find where to put generate a PAT for the organization

Reply:
I think [Automatic token authentication](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) should do the trick, which is what I quoted from (more or less). Wether it works or not, I don't know. Maybe something to try for another time.